src/storage/storage-wrapper.js

```diff
@@ -0,0 +1,21 @@
+import tokens from '@aragon/templates-tokens'
+import { soliditySha3 } from '../web3-utils'
+import storageAbi from './storage-abi.json'
+
+export function testTokensEnabled(network) {
+  return !!tokens[network]
+}
+const getContract = (web3, storageAddr) => {
+  return new web3.eth.Contract(storageAbi, storageAddr)
+}
```

Comment:
We can just define a constant for this, as seen in airdrop.js

Reply:
I think using a constant is not possible due to storageAddr variable is unknown until it loads. To avoid this we can store it in some sort of cache that can initialize when needed.
Since airdrop.js returns a function, i think that constant is being initialized every time you execute it.